src/sql/workbench/test/electron-browser/api/vscodeNotebookApi.test.ts

```diff
@@ -440,6 +445,22 @@ suite('Notebook Serializer', () => {
 		secondCell = vsDoc.cellAt(10);
 		validateCellMatches(secondCell, expectedCells[1]);
 	});
+
+	test('VS Code NotebookEditor functionality', async () => {
+		let editor = <azdata.nb.NotebookEditor>{ document: testDoc };
+		let vscodeEditor = new VSCodeNotebookEditor(editor);
+		let expectedDoc = new VSCodeNotebookDocument(testDoc);
+
+		validateDocsMatch(vscodeEditor.document, expectedDoc);
+
+		// We only need the document field for VSCodeNotebookEditor, so the other
+		// fields should be non-functional
```

Comment:
Do we not want to throw an error for these ones to clearly indicate to the user that they're not valid? It would be unfortunate to have people accessing these and being confused when it isn't returning the "correct" values